src/main/resources/wa-task-permissions-ia-asylum.dmn

```diff
@@ -501,6 +501,156 @@ else
           <text>false</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_14kbksv">
```

Comment:
what about case-manager and task-supervisor? I can't see any documentation that explicitly verifies the role-names mapped?